### PR TITLE
Simplifies the "All we want to do..." portion

### PR DESCRIPTION
**Pull request by zachgarwood**
On the Routes & Templates page, in the An Index Route section, the
paragraphs starting at "All we want to do..." present too much
ancillary information. This change simplifies it to focus on just the
beforeModel() hook and the replaceWith() function.

This addresses one of the "tiny tutorial fixes" @jenweber listed
in #245.

### CODE REVIEW

File: guides/v3.5.0/tutorial/routes-and-templates.md [hunk 1]
```diff
@@ -219,22 +219,11 @@ Unlike the other route handlers we've made so far, the `index` route is special:
...
-In our index route handler, we'll call the [`replaceWith`](https://www.emberjs.com/api/ember/release/classes/Route/methods/beforeModel?anchor=replaceWith) function.
-The `replaceWith` function is similar to the route's [`transitionTo()`](https://www.emberjs.com/api/ember/release/classes/Route/methods/transitionTo?anchor=transitionTo) function,
-the difference being that `replaceWith` will replace the current URL in the browser's history,
-while `transitionTo` will add to the history.
-Since we want our `rentals` route to serve as our home page, we will use the `replaceWith` function.
-
-In our index route handler, we add the `replaceWith` invocation to `beforeModel`.
+All we want to do when a user visits the root (`/`) URL is transition to
+`/rentals`. To do this we will add code to our index route handler by
+implementing a route lifecycle hook called `beforeModel`. Inside, we'll call the
```
Comment (jenweber) [inline guides/v3.5.0/tutorial/routes-and-templates.md]:
```suggestion
implementing a route lifecycle hook called `beforeModel`. 
Route lifecycle hooks are special methods that are called automatically when a route renders or data changes.
Inside, we'll call the
```

We should still give a super brief description of a lifecycle hook. This is an example but I am not 100% confident that it's technically accurate. What do you think?

**Comment by zachgarwood** [comment]
@jenweber I think your suggestion was succinct and provides exactly as much context as needed, namely that hooks are called automatically at certain events.

**Comment by jenweber** [comment]
Cool, thanks! Merged.

**Comment by jenweber** [comment]
Note to self, this has been added to the 3.6 staging.

**Pull request merged**